org.linux_hardware.hw-probe.yaml

```diff
@@ -0,0 +1,393 @@
+      - make install DESTDIR=`pwd`/INST INSTALLSITELIB=/SITELIB
+      - mkdir -p $FLATPAK_DEST/usr/share/perl5/
+      - cp -fr ./INST/SITELIB/* $FLATPAK_DEST/usr/share/perl5/
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/G/GB/GBARR/TimeDate-2.30.tar.gz
+        sha256: 75bd254871cb5853a6aa0403ac0be270cdd75c9d1b6639f18ecba63c15298e86
+  - name: libwww-perl
+    buildsystem: simple
+    build-commands:
+      - perl Makefile.PL
+      - make install DESTDIR=`pwd`/INST INSTALLSITELIB=/SITELIB
+      - mkdir -p $FLATPAK_DEST/usr/share/perl5/
+      - cp -fr ./INST/SITELIB/* $FLATPAK_DEST/usr/share/perl5/
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/E/ET/ETHER/libwww-perl-6.35.tar.gz
+        sha256: dda2578d7b32152c4afce834761a61d117de286c705a9f7972c7ac6032ca5953
+  - name: perl-parent
+    buildsystem: simple
+    build-commands:
+      - perl Makefile.PL
+      - make install DESTDIR=`pwd`/INST INSTALLSITELIB=/SITELIB
+      - mkdir -p $FLATPAK_DEST/usr/share/perl5/
+      - cp -fr ./INST/SITELIB/* $FLATPAK_DEST/usr/share/perl5/
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/C/CO/CORION/parent-0.237.tar.gz
+        sha256: 1089d9648565c1d1e655fa4cb603272d3126747b7b5f836ffee685e27e53caae
+  - name: perl-time-local
+    buildsystem: simple
+    build-commands:
+      - perl Makefile.PL
+      - make install DESTDIR=`pwd`/INST INSTALLSITELIB=/SITELIB
+      - mkdir -p $FLATPAK_DEST/usr/share/perl5/
+      - cp -fr ./INST/SITELIB/* $FLATPAK_DEST/usr/share/perl5/
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Time-Local-1.28.tar.gz
+        sha256: 9278b9e5cc99dcbb0fd27a43e914828b59685601edae082889b5ee7266afe10e
+  - name: perl-data-dumper
+    buildsystem: simple
+    build-commands:
+      - perl Makefile.PL
+      - make install DESTDIR=`pwd`/INST INSTALLSITELIB=/SITELIB
+      - chmod 777 -R ./INST
+      - find ./INST -type f | perl -lne 'print if -B and -x' | xargs strip
+      - mkdir -p $FLATPAK_DEST/usr/share/perl5/
+      - cp -fr ./INST/usr/lib/perl/*/* $FLATPAK_DEST/usr/share/perl5/
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/S/SM/SMUELLER/Data-Dumper-2.161.tar.gz
+        sha256: 3aa4ac1b042b3880438165fb2b2139d377564a8e9928ffe689ede5304ee90558
+  - name: hw-probe
+    buildsystem: simple
+    build-commands:
+      - make install DESTDIR=$FLATPAK_DEST
+      - mkdir -p $FLATPAK_DEST/bin
+      - mv flatpak/hw-probe.sh $FLATPAK_DEST/bin/hw-probe-flatpak
+      - mv flatpak/PERL5_BASE $FLATPAK_DEST/usr/share/
+      - chmod a+x $FLATPAK_DEST/bin/hw-probe-flatpak
+      - mkdir -p $FLATPAK_DEST/share/icons/hicolor/256x256/apps $FLATPAK_DEST/share/icons/hicolor/128x128/apps $FLATPAK_DEST/share/icons/hicolor/64x64/apps $FLATPAK_DEST/share/applications $FLATPAK_DEST/share/metainfo $FLATPAK_DEST/usr/share/
```

Comment:
You could just use `install` instead of creating the folders manually. Plus, why you would need `$FLATPAK_DEST/usr/share` ? the app data files must be in `/app/share`

Reply:
This is fixed by one of the previous commits.